packages/augur-ui/src/modules/market/components/market-view/market-view.tsx

```diff
@@ -597,7 +611,7 @@ export default class MarketView extends Component<
 
                       <TradingForm
                         market={market}
-                        initialLiquidity={preview && !initialLiquidity}
+                        initialLiquidity={preview && !tradingTutorial}
```

Comment:
what happen with `!initialLiquidity` boolean

Reply:
this should prob be initialLiquidity || (preview && !tradingTutorial)

Reply:
we don't need it here because the market creation doesn't use the market view